apps/tangle-dapp/data/liquidStaking/getValueOfTangleCurrency.ts

```diff
@@ -3,51 +3,47 @@
 import '@webb-tools/tangle-restaking-types';
 
 import { TanglePrimitivesCurrencyCurrencyId } from '@polkadot/types/lookup';
+import { capitalize } from 'lodash';
```

Comment:
Use  
```tsx
import capitalize from 'lodash/capitalize';
```  
to optimize tree-shaking and avoid importing the entire `lodash` library.